snuba/query/extensions.py

```diff
@@ -0,0 +1,65 @@
+from abc import ABC
+from datetime import timedelta
+
+from snuba.query.query_processor import (
+    DummyExtensionProcessor,
+    ExtensionQueryProcessor
+)
+from snuba.schemas import (
+    get_time_series_extension_properties,
+    PERFORMANCE_EXTENSION_SCHEMA,
```

Comment:
@tkaemming, I would move the extension schemas close to the extensions (which means in this file). Do you see any reason not to do that ?

Reply:
I like the idea, fwiw.

Reply:
I think it's a good idea to keep the schema definitions as close as possible to the objects that they represent/that use them, so yeah, I think this is a good idea.

I might even suggest moving the generic query schema into `snuba.query.query` since you mentioned earlier you think that this will trend towards becoming the Snuba query representation (rather than the ClickHouse one.) The structure of this module seems to reinforce this as well, since extensions are siblings in a hierarchical sense.